compile_upload.R

```diff
@@ -1,50 +1,48 @@
 ## this script is for pre-compiling data and running/uploading app
 
 # set order of read-in scripts
-read_in_scripts <- c(
-  "clean_liquid.R", "seq.R", "rna_seq.R", "oncoprint2.R",
-  "clean_solid.R", "jax.R", "finalize.R"
-)
+read_in_scripts = c("clean_liquid.R", "clean_virusseq.R", "seq.R","rna_seq.R","oncoprint2.R",
```

Comment:
@sharlagelfand : I'm noticing that your nicely-spaced reformatting seems to be being overwritten all through this document. How do we want to fix/avoid this?

Reply:
Thanks, I just reapplied the styling. I think it was because I was working in various branches, but I'll make sure files are styled going forward.